hedera-node/hedera-app/build.gradle.kts

```diff
@@ -146,25 +145,6 @@ val copyLib =
     tasks.register<Sync>("copyLib") {
         from(project.configurations.getByName("runtimeClasspath"))
         into(layout.projectDirectory.dir("../data/lib"))
-
-        doLast {
-            val nonModulalJars =
-                destinationDir
-                    .listFiles()!!
-                    .mapNotNull { jar ->
-                        if (zipTree(jar).none { it.name == "module-info.class" }) {
-                            jar.name
-                        } else {
-                            null
-                        }
-                    }
-                    .sorted()
-            if (nonModulalJars.isNotEmpty()) {
-                throw RuntimeException(
-                    "Jars without 'module-info.class' in 'data/lib'\n${nonModulalJars.joinToString("\n")}"
-                )
-            }
-        }
```

Comment:
The way I did this (in #8561), is not compatible with the configuration cache. I will re-add a similar check in a follow up. (By now the `extra-java-module-info` has some new functionality –  https://github.com/gradlex-org/extra-java-module-info/pull/75 – that may help us.)